public/app/percona/dbaas/components/DBCluster/AddDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -89,33 +133,74 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({ values, form })
           />
         )}
       </div>
-      <RadioButtonGroupField
-        name={AddDBClusterFields.resources}
-        label={Messages.dbcluster.addModal.fields.resources}
-        options={RESOURCES_OPTIONS}
-      />
-      <div className={styles.resourcesWrapper}>
-        <NumberInputField
-          name={AddDBClusterFields.memory}
-          label={Messages.dbcluster.addModal.fields.memory}
-          validators={parameterValidators}
-          disabled={resources !== DBClusterResources.custom}
-          parse={parseNonNegativeFloat}
-        />
-        <NumberInputField
-          name={AddDBClusterFields.cpu}
-          label={Messages.dbcluster.addModal.fields.cpu}
-          validators={parameterValidators}
-          disabled={resources !== DBClusterResources.custom}
-          parse={parseNonNegativeFloat}
-        />
-        <NumberInputField
-          name={AddDBClusterFields.disk}
-          label={Messages.dbcluster.addModal.fields.disk}
-          validators={diskValidators}
-          disabled={resources !== DBClusterResources.custom}
-          parse={parsePositiveInt}
+      <div className={styles.resourcesRadioWrapper}>
+        <RadioButtonGroupField
+          name={AddDBClusterFields.resources}
+          label={Messages.dbcluster.addModal.fields.resources}
+          options={RESOURCES_OPTIONS}
         />
+        <div className={styles.resourcesInfoWrapper}>
+          <Icon className={styles.resourcesInfoIcon} name="info-circle" />
+          <span>{Messages.dbcluster.addModal.resourcesInfo}</span>
+        </div>
+      </div>
+      <div className={styles.resourcesWrapper}>
+        <div className={styles.resourcesInputCol}>
+          <NumberInputField
+            name={AddDBClusterFields.memory}
+            label={Messages.dbcluster.addModal.fields.memory}
+            validators={parameterValidators}
+            disabled={resources !== DBClusterResources.custom}
+            parse={parseNonNegativeFloat}
+          />
+          <NumberInputField
+            name={AddDBClusterFields.cpu}
+            label={Messages.dbcluster.addModal.fields.cpu}
+            validators={parameterValidators}
+            disabled={resources !== DBClusterResources.custom}
+            parse={parseNonNegativeFloat}
+          />
+          <NumberInputField
+            name={AddDBClusterFields.disk}
+            label={Messages.dbcluster.addModal.fields.disk}
+            validators={diskValidators}
+            disabled={resources !== DBClusterResources.custom}
+            parse={parsePositiveInt}
+          />
+        </div>
+        <div className={styles.resourcesBarCol}>
+          <Overlay isPending={loadingResources}>
+            <ResourcesBar
+              resourceLabel={Messages.dbcluster.addModal.resourcesBar.memory}
+              icon={<Memory />}
+              total={allocatedResources?.total.memory}
+              allocated={allocatedResources?.allocated.memory}
+              expected={undefined}
```

Comment:
if `expected` is not passed, doesn't that make it `undefined`?

Reply:
This is just temporary, will be changed sooner in the expected resources feature.

Reply:
👍